java/tools/src/java/org/apache/orc/tools/ColumnSizes.java

```diff
@@ -217,6 +227,11 @@ private static Options createOptions() {
         .desc("Ignore ORC file extension")
         .build());
 
+    result.addOption(Option.builder("s")
+        .longOpt("summary")
+        .desc("Summarize the number of files, file size, and number of file lines")
```

Comment:
`file lines`? `Line` is not Apache ORC format. Maybe, did you want to say `Rows`?

Reply:
Thanks. Updated.